providers/src/airflow/providers/edge/executors/edge_executor.py

```diff
@@ -129,6 +129,34 @@ def execute_async(
             )
         )
 
+    @provide_session
+    def queue_workload(
+        self,
+        workload: Any,  # Note actually "airflow.executors.workloads.All" but not existing in Airflow 2.10
```

Comment:
Do we need to keep this edge executor provider working with 2.0, or should/could we make use of this need Airflow 3?

Reply:
For the moment we are using Edge Executor in Airflow 2.10. Once we have Airflow 3 as GA, the agreement was to drop Airflow 2.10 support. Then we can clean-up the type definition (and other compat things)